public/styles/main.css

```diff
@@ -144,7 +144,54 @@
   to {
     background-color: lime; } }
 
+#deathAnimation {
+  background-image: url(/img/death0.png);
+  background-size: 200px;
+  background-repeat: no-repeat;
+  position: absolute;
+  z-index: 1;
+  left: 600px;
+  top: 350px;
+  width: 250px;
+  height: 250px;
+  overflow: auto;
+  display: none;
+  animation-delay: 4s;
+  animation-fill-mode: forwards;
+  animation: jumpscare .5s;
+  animation-timing-function: linear;
+  animation-direction: alternate; }
+
+@keyframes jumpscare {
+  0% {
+    transform: rotate(0deg); }
```

Comment:
Don't forget to add prefixes for transformations
-webkit-transform, -moz-transform